src/DssSpell.t.sol

```diff
@@ -18,36 +18,26 @@ pragma solidity 0.6.12;
 
 import "./DssSpell.t.base.sol";
 
-interface RwaLiquidationLike {
-    function ilks(bytes32) external returns (string memory, address, uint48 toc, uint48 tau);
-    function bump(bytes32 ilk, uint256 val) external;
-    function tell(bytes32) external;
-    function cure(bytes32) external;
-    function cull(bytes32, address) external;
-    function good(bytes32) external view returns (bool);
+interface ERC20Like {
+    function balanceOf(address) external view returns (uint256);
 }
 
-interface RwaUrnLike {
-    function can(address) external view returns (uint256);
-    function lock(uint256) external;
-    function draw(uint256) external;
-    function wipe(uint256) external;
-    function free(uint256) external;
-}
+contract DssSpellTest is DssSpellTestBase {
 
-interface RwaOutputConduitLike {
-    function can(address) external view returns (uint256);
-    function may(address) external view returns (uint256);
-    function pick(address) external;
-    function push() external;
-}
+    function test_RWA009Draw() public {
```

Comment:
Maybe I'd suggest to put this week specific test at the end of the file, replacing where last week RWAs ones were.

Reply:
done